styx-common/src/main/java/com/spotify/styx/testdata/TestData.java

```diff
@@ -140,6 +143,37 @@
           .retryCondition("#exitCode == 1 && (#tries < 3 || #consecutiveFailures < 4) && #triggerType == \"natural\"")
           .build();
 
+  public static final WorkflowConfiguration DOCKER_EXEC_WORKFLOW_CONFIGURATION =
+      WorkflowConfiguration.builder()
+          .id("styx.TestEndpoint")
+          .commitSha(VALID_SHA)
+          .schedule(DAYS)
+          .serviceAccount("foo@bar.baz.quux")
+          .dockerExecConf(new DockerExecConfBuilder()
```

Comment:
Why do we have a 
WorkflowConfiguration.builder(), but then a `new DockerExecConfBuilder()` 

Should we be consistent with what we use?